payment/controllers.go

```diff
@@ -30,6 +31,126 @@ func Router(service *Service) chi.Router {
 	return r
 }
 
+// KeyRouter handles management of keys
+func KeyRouter(service *Service) chi.Router {
+	r := chi.NewRouter()
+	if os.Getenv("ENV") != "local" {
+		r.Use(middleware.SimpleTokenAuthorizedOnly)
+	}
+
+	r.Method("GET", "/{merchantId}", middleware.InstrumentHandler("GetKeys", GetKeys(service)))
+	r.Method("POST", "/", middleware.InstrumentHandler("CreateKey", CreateKey(service)))
+	r.Method("DELETE", "/{id}", middleware.InstrumentHandler("DeleteKey", DeleteKey(service)))
```

Comment:
I am not sure I like that the GET URL parameter is not the same as the DELETE URL parameter.  I would rather get the `merchantId` value from the authentication token that is used to authenticate the merchant and have no GET url parameter to return all keys for that merchant.

Reply:
Realized this is called by a whitelisted app.